sample/host/webpack.dev.js

```diff
@@ -0,0 +1,66 @@
+// @ts-check
+
+// Added for TSC, otherwise the "devServer" section is unknown.
+import "webpack-dev-server";
+
+import HtmlWebpackPlugin from "html-webpack-plugin";
+import ReactRefreshWebpackPlugin from "@pmmmwh/react-refresh-webpack-plugin";
+import { fileURLToPath } from "url";
+import path from "path";
+
+/** @type {import("webpack").Configuration} */
+export default {
+    mode: "development",
+    target: "web",
+    devtool: "inline-source-map",
```

Comment:
Webpack recommends `eval-cheap-module-source-map` for development because it's faster.